corehq/apps/users/device_rate_limiter.py

```diff
@@ -7,6 +8,9 @@
 from corehq.apps.cloudcare.const import DEVICE_ID as CLOUDCARE_DEVICE_ID
 from corehq.util.metrics import metrics_counter, metrics_histogram
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
I think this INFO level setting should go with logging configs in _settings.py_. Otherwise I think it is impossible to override the logging level in _localsettings.py_.

Reply:
963cb4275a469669a08dd7ce1056e3f99d39057e